SpokestackTests/TextToSpeechTest.swift

```diff
@@ -32,6 +32,17 @@ class TextToSpeechTest: XCTestCase {
         let config = SpeechConfiguration()
         let tts = TextToSpeech(delegate, configuration: config)
         
+        // bad input results in a failed request that calls failure
+        delegate.reset()
+        let didFailInputExpectation = expectation(description: "bad input results in a failed request that calls failure")
+        delegate.asyncExpectation = didFailInputExpectation
+        let badInput = TextToSpeechInput()
+        badInput.voice = "tracy-throne"
+        tts.synthesize(badInput)
+        wait(for: [didFailInputExpectation], timeout: 5)
+        XCTAssert(delegate.didFail)
+        XCTAssertFalse(delegate.didSucceed)
```

Comment:
I'm not entirely sure how the waiting thing works here, but thisisfine. Might want to test how your error handling code works with actual graphql error responses (see my problem w/Android), but I get the annoyance there.

Reply:
hah, knew you'd chime in with that. if this were BDD or something I might, but the mechanism isn't important enough. The GraphQL error was being thrown correctly before these changes, the changes are just to make the error more informative.